src/extensionsIntegrated/CSSColorPreview/main.js

```diff
@@ -142,6 +144,45 @@ define(function (require, exports, module) {
         });
     }
 
+
+    /**
+     * To move the cursor to the color text and display the color quick edit
+     * @param {Editor} codem the codemirror instance
+     * @param {Number} lineNumber the line number that is clicked
+     * @param {String} gutter the gutter name
+     */
+    function colorIconClicked(codem, lineNumber, gutter) {
+        const editor = EditorManager.getActiveEditor();
+        const cm = editor._codeMirror;
+
+        if(gutter === 'CodeMirror-linenumbers') {
```

Comment:
The color click was not working as the gutter name here should be for color gutter. fixed it in https://github.com/phcode-dev/phoenix/pull/1965